frontend/packages/dev-console/src/components/project-access/__tests__/ProjectAccessForm.spec.tsx

```diff
@@ -0,0 +1,226 @@
+        ],
+      },
+      isValid: true,
+      registerField: jest.fn(),
+      resetForm: jest.fn(),
+      setErrors: jest.fn(),
+      setFieldError: jest.fn(),
+      setFieldTouched: jest.fn(),
+      setFieldValue: jest.fn(),
+      setFormikState: jest.fn(),
+      setStatus: jest.fn(),
+      setSubmitting: jest.fn(),
+      setTouched: jest.fn(),
+      setValues: jest.fn(),
+      submitForm: jest.fn(),
+      unregisterField: jest.fn(),
+      validateField: jest.fn(),
+      validateForm: jest.fn(),
+      validateOnBlur: true,
+      validateOnChange: true,
+    };
+  });
+  it('should load the Project Access Form', () => {
+    const projectAccessForm = shallow(<ProjectAccessForm {...formProps} />);
+    expect(projectAccessForm.find(Form).exists()).toBeTruthy();
+    expect(
+      projectAccessForm
+        .find(Form)
+        .shallow()
+        .find(MultiColumnField)
+        .exists(),
+    ).toBeTruthy();
+    expect(
+      projectAccessForm
+        .find(Form)
+        .shallow()
+        .find(MultiColumnField)
+        .props().name,
+    ).toBe('projectAccess');
+    expect(
+      projectAccessForm
+        .find(Form)
+        .shallow()
+        .find(MultiColumnField)
+        .props().toolTip,
+    ).toBe('Remove Access');
+    expect(
+      projectAccessForm
+        .find(Form)
+        .shallow()
+        .find(MultiColumnField)
+        .props().headers,
+    ).toEqual(['Name', 'Role']);
+    expect(
+      projectAccessForm
+        .find(Form)
+        .shallow()
+        .find(MultiColumnField)
+        .children(),
+    ).toHaveLength(2);
+    expect(
+      projectAccessForm
+        .find(Form)
+        .shallow()
+        .find(MultiColumnField)
+        .children()
+        .at(0)
+        .is(InputField),
+    ).toBeTruthy();
+    expect(
+      projectAccessForm
+        .find(Form)
+        .shallow()
+        .find(MultiColumnField)
+        .children()
+        .at(1)
+        .is(DropdownField),
+    ).toBeTruthy();
+    expect(
+      projectAccessForm
+        .find(Form)
+        .shallow()
+        .find(MultiColumnField)
+        .props().addLabel,
+    ).toEqual('Add Access');
+    expect(
+      projectAccessForm
+        .find(Form)
+        .shallow()
+        .find(FormFooter)
+        .exists(),
+    ).toBeTruthy();
+  });
+
+  it('should load the dropdown with access roles', () => {
+    const projectAccessForm = shallow(<ProjectAccessForm {...formProps} />);
+    expect(
+      projectAccessForm
+        .find(Form)
+        .shallow()
+        .find(MultiColumnField)
+        .children()
+        .at(1)
+        .props().name,
+    ).toBe('role');
+    expect(
+      projectAccessForm
+        .find(Form)
+        .shallow()
+        .find(MultiColumnField)
+        .children()
+        .at(1)
+        .props().items,
+    ).toEqual({ '': 'Select a role', admin: 'Admin', view: 'View', edit: 'Edit' });
+  });
+
+  it('should have the save and reload buttons ', () => {
```

Comment:
You're testing the `FormFooter` component, not yours. Drop this test as it doesn't test anything your component has the power over.

You could replace it with a simple "Renders `FormFooter`" test, but I wouldn't go any further than that.